data/json/recipes/recipe_others.json

```diff
@@ -3722,7 +3722,8 @@
      [ "steel_lump", 1 ],
      [ "steel_chunk", 4 ],
      [ "scrap", 12 ],
-     [ "pipe", 3 ]
+     [ "pipe", 3 ],
+     [ "material_aluminium_ingot", 1 ]
```

Comment:
That item is still made of `steel`. We probably shouldn't mix materials like this